docs/pages/architecture/trustedclusters.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Teleport Trusted Clusters
+description: Deep dive into design of Teleport Trusted Clusters.
+h1: Trusted Clusters Architecture
+---
+
+## Overview
+
+Teleport can partition compute infrastructure into multiple clusters. A cluster
+is a group of Teleport resources connected. Each clusters
+manages a separate certificate authority (CA) for all users and resources.
```

Comment:
Technically each cluster manages several CAs.

Could we say "each cluster issues certificates for users and other resources in the cluster" instead?